src/index.js

```diff
@@ -0,0 +1,165 @@
+      document.getElementById('range').value = fTemp;
+      changeColors();
+    })
+    .catch((error) => {
+      console.log('error');
+    });
+};
+
+function clickBtn(event) {
+  const city = document.getElementById('city').value;
+  state.location = city;
+  getLonLat();
+}
+
+const inputBox = document.getElementById('city');
+inputBox.onkeyup = function () {
+  document.getElementById('city-name').innerHTML = inputBox.value;
+};
+
+function rangeSlide(event) {
+  const value = event.currentTarget.value;
+  document.getElementById('rangeValue').textContent = value;
+}
+
+const validateSkyCondition = (condition) => {
+  console.log(condition);
+  if (condition === 'Clear') {
+    state.sky.value = 'sunny';
+    document.getElementById('skySelect').value = '#7496C9';
+  } else if (
+    condition === 'Rain' ||
+    condition === 'Drizzle' ||
+    condition === 'Thunderstorm'
+  ) {
+    state.sky.value = 'rainy';
+    document.getElementById('skySelect').value = '#356098';
+  } else if (condition === 'Clouds' || condition === 'Haze') {
+    state.sky.value = 'cloudy';
+    document.getElementById('skySelect').value = '#557BB1';
+  } else if (condition === 'Snow') {
+    state.sky.value = 'snowy';
+    document.getElementById('skySelect').value = '#234772';
+  }
+  skyChange();
+};
+
+const changeColors = () => {
+  const rangeVal = document.getElementById('range');
+  if (rangeVal.value >= 80) {
+    document.getElementById('city-circle').style.background = '#EA9A9F';
+    document.getElementById('temp-circle').style.background = '#EA9A9F';
+    document.getElementById('sky-circle').style.background = '#EA9A9F';
+    document.getElementById('weather').style.color = '#EA9A9F';
+  } else if (rangeVal.value >= 70) {
+    document.getElementById('city-circle').style.background = '#E8BCC3';
+    document.getElementById('temp-circle').style.background = '#E8BCC3';
+    document.getElementById('sky-circle').style.background = '#E8BCC3';
+    document.getElementById('weather').style.color = '#E8BCC3';
+  } else if (rangeVal.value >= 60) {
+    document.getElementById('city-circle').style.background = '#E5DDE7';
+    document.getElementById('temp-circle').style.background = '#E5DDE7';
+    document.getElementById('sky-circle').style.background = '#E5DDE7';
+    document.getElementById('weather').style.color = '#E5DDE7';
+  } else if (rangeVal.value >= 50) {
+    document.getElementById('city-circle').style.background = '#CECEE0';
+    document.getElementById('temp-circle').style.background = '#CECEE0';
+    document.getElementById('sky-circle').style.background = '#CECEE0';
+    document.getElementById('weather').style.color = '#CECEE0';
+  } else if (rangeVal.value < 50) {
+    document.getElementById('city-circle').style.background = '#AEBFDD';
+    document.getElementById('temp-circle').style.background = '#AEBFDD';
+    document.getElementById('sky-circle').style.background = '#AEBFDD';
+  }
+};
```

Comment:
Hm... `changeColors` and `changeRangeS` are very similar. There's probably a way of combining both of them!